infra/charts/feast-feature-server/values.yaml

```diff
@@ -38,6 +40,11 @@ service:
   type: ClusterIP
   port: 80
 
+otel:
+  exporter:
+    endpoint: "http://otel-collector.feast-test.svc.cluster.local:4317"
+    insecure: "true"
```

Comment:
this needs to be talked through... this endpoint should be more dynamically set correct?